tcp_check/datadog_checks/tcp_check/config_models/defaults.py

```diff
@@ -16,6 +16,10 @@ def instance_empty_default_hostname(field, value):
     return False
 
 
+def instance_ip_cache_duration(field, value):
+    return 120
```

Comment:
@ofek Shouldn't this be null?

Reply:
This was generated by `ddev validate models -s`; I believe it is taking the value from `example` rather than `display_default`. I found where I need to fix it (here in https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/specs/configuration/consumers/model.py#L37-L38) but I think it would be best to do it in another PR since it would affect other integrations.